docs/API.md

```diff
@@ -194,6 +194,14 @@ renameProp(
 
 Renames a single prop.
 
+NOTE: We recommend using `#withProps` and ES6 syntax instead. This method may be deprecated in future releases.
+
+```
+renameProp('oldName', 'newName')
+->
+withProps(props => ({newName: props.oldName}))
```

Comment:
We should follow the style guide. Please add spaces after `{` and before `}`.

e.g.

```js
withProps(props => ({ newName: props.oldName }))
````